.github/workflows/maven-publish.yml

```diff
@@ -0,0 +1,34 @@
+# This workflow will build a package using Maven and then publish it to GitHub packages when a release is created
+# For more information see: https://github.com/actions/setup-java/blob/main/docs/advanced-usage.md#apache-maven-with-a-settings-path
+
+name: Maven Package
```

Comment:
This workflow in particular should be removed for merge

Reply:
Okaay